test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -335,4 +336,68 @@ def add_locations
     assert_response :success
     assert_equal 1, location.reload.location_parameters.count
   end
+
+  test "should create with comma separated name" do
+    name = "%s, %s" % [RFauxFactory.gen_alpha, RFauxFactory.gen_alpha]
+    post :create, params: { :location => { :name => name } }
+    assert_response :success
+    show_response = JSON.parse(@response.body)
+    assert !show_response.empty?
```

Comment:
this would be better written as `refute_empty show_response`, it would avoid negation and print better error message if the assertion fails

Reply:
the same applies in the test below